playwright-e2e/pages/orders.ts

```diff
@@ -48,19 +60,34 @@ export class Orders extends BasePage {
         await this.orderTypeRadio.getByLabel(`${orderType}`).check();
     }
 
-    async addIssuingDetails() {
+    async addIssuingDetailsOfApprovedOrder() {
         await this.orderApproved.getByLabel('Yes').click();
         await this.approvedHearing.selectOption('Case management hearing, 3 November 2012');
         await this.orderApplication.getByLabel('No').click();
         await this.clickContinue();
         await this.issuingJudge.getByRole('radio', {name: 'Yes'}).check();
     }
+    async addIssuningDeatilsOfUnApprovedOrder(){
```

Comment:
just minor - typo in function name, should  be - addIssuingDetailsOfUnApprovedOrder